src/components/DataTable/README.md

```diff
@@ -306,3 +306,54 @@ Keep decimals consistent. For example, don’t use 3 decimals in one row and 2 i
 ## Related components
 
 - To create an actionable list of related items that link to details pages, such as a list of customers, use the [resource list component](/components/lists-and-tables/resource-list).
+
+---
+
+## Accessibility
+
+<!-- content-for: android -->
+
+See Material Design and development documentation about accessibility for Android:
+
+- [Accessible design on Android](https://material.io/design/usability/accessibility.html)
+- [Accessible development on Android](https://developer.android.com/guide/topics/ui/accessibility/)
+
+<!-- /content-for -->
+
+<!-- content-for: ios -->
+
+See Apple’s Human Interface Guidelines and API documentation about accessibility for iOS:
+
+- [Accessible design on iOS](https://developer.apple.com/design/human-interface-guidelines/ios/app-architecture/accessibility/)
+- [Accessible development on iOS](https://developer.apple.com/accessibility/ios/)
+
+<!-- /content-for -->
+
+<!-- content-for: web -->
+
+### Structure
+
+Native HTML tables provide a large amount of structural information to screen reader users. Merchants who rely on screen readers can navigate tables and identify relationships between data cells (`<td>`) and headers (`<th>`) using keys specific to their screen reader.
+
+Sortable tables use the `aria-sort` attribute to convey which columns are sortable (and in what direction). They also use `aria-label` on sorting buttons to convey what activating the button will do.
+
+<!-- usageblock -->
+
+#### Do
+
+Use tables for tabular data.
+
+#### Don’t
+
+Use tables for layout. For a table-like layout that doesn’t use table HTML elements, use the [resource list component](/components/lists-and-tables/resource-list).
+
+<!-- end -->
+
+### Keyboard support
+
+Sorting controls for the data table component are implemented with native HTML buttons.
+
```

Comment:
Note on the list below: it describes behaviors that should come for free when using native HTML buttons (as recommended above). Wondering if this is still worth mentioning?

Reply:
I'd like to include this because folks don't always know what keyboard inputs are native.